homeassistant/components/ruuvi_gateway/__init__.py

```diff
@@ -0,0 +1,45 @@
+"""The Ruuvi Gateway integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_TOKEN, Platform
+from homeassistant.core import HomeAssistant
+
+from .bluetooth import async_connect_scanner
+from .const import DOMAIN
+from .coordinator import RuuviGatewayUpdateCoordinator
+from .models import RuuviGatewayRuntimeData
+
+PLATFORMS: list[Platform] = []
+
+_LOGGER = logging.getLogger(DOMAIN)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Ruuvi Gateway from a config entry."""
+    coordinator = RuuviGatewayUpdateCoordinator(
+        hass,
+        logger=_LOGGER,
+        name=entry.title,
+        update_interval=timedelta(seconds=10),
```

Comment:
Any reason to not poll at 5s?

Reply:
Not really, I can change that if 5s is a standard.